.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           cache: yarn
       - uses: dtolnay/rust-toolchain@master
         with:
-          toolchain: 1.77
+          toolchain: stable
```

Comment:
I'm not sure if we should stay on a pinned release of Rust, and update it manually? Or just stay on `stable` and pin only when it breaks?

Reply:
> Or just stay on stable and pin only when it breaks?

Yes, that's what I would have suggested (= what we've done so far). Requires less maintenance from our side as well